SPECS/unbound/unbound.spec

```diff
@@ -1,6 +1,6 @@
 Summary:        unbound dns server
 Name:           unbound
-Version:        1.16.3
```

Comment:
We normally don't take minor-version upgrades for CVEs -- just patch-version upgrades. Have we done due diligence on this?

Reply:
Great catch, @tobiasb-ms! I've just ran a local and [buddy build](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=516379&view=results) for the updated `unbound` and two packages, which depend on it (fortunately that's all there is). All have package tests and all still pass after the update, so in this case, I'd say we can allow a minor-version upgrade.